### ISSUE TITLE
!important removed when auto prefixed

### ISSUE THREAD

**Issue by kibs** [styled-components/styled-components#620]
## Version
2.0.0-8

## Reproduction

Sorry could not get webpackbin to work

## Steps to reproduce
```css
display: flex !important;
```

## Expected Behavior
```css
display: -webkit-box !important;
display: -webkit-flex !important;
display: -ms-flexbox !important;
display: flex !important;
```

## Actual Behavior
```css
display: -webkit-box;
display: -webkit-flex;
display: -ms-flexbox;
display: flex;
```

**Comment by thysultan** [comment]
Thanks, fixed v1.2.8

**Comment by kibs** [comment]
Wow, you work fast!
